noxfile.py

```diff
@@ -6,6 +6,18 @@
 import shutil
 import nox
 
+@nox.session(py="3")
+def translation(session):
+    session.install("sphinx-intl")
+    session.install("sphinx-inline-tabs")
+    target_dir = "locales"
+    session.run(
+            "sphinx-build", 
+            "-b",  # select a builder
+            "gettext",  # build gettext-style message catalogs (.pot file)
+            "source/",  # where the rst files are located
+            target_dir, # where to put the .pot file
+        )
```

Comment:
Let's align with the style of the rest of this Python module:
```suggestion
        "sphinx-build", 
        "-b",  # select a builder
        "gettext",  # build gettext-style message catalogs (.pot file)
        "source/",  # where the rst files are located
        target_dir, # where to put the .pot file
    )
```